spec-main/autofill-spec.ts

```diff
@@ -0,0 +1,42 @@
+import { BrowserWindow } from 'electron';
+import * as path from 'path';
+import { delay } from './spec-helpers'
+import { expect } from 'chai'
+
+const fixturesPath = path.resolve(__dirname, '..', 'spec', 'fixtures');
+
+describe('autofill', () => {
+  let w: BrowserWindow = null as unknown as BrowserWindow;
+
+  beforeEach(async () => {
+    w = new BrowserWindow({
+      show: true,
+      webPreferences: {
+        nodeIntegration: true,
+      }
+    })
+
+    await w.loadFile(path.join(fixturesPath, 'pages', 'datalist.html'))
+  })
+
+  afterEach(() => {
+    w.destroy()
+    w = null as unknown as BrowserWindow
+  })
```

Comment:
I recommend the "new style" of test fixture, which is to create browserwindows in your test as needed and add `afterEach(closeAllWindows)` as cleanup.

Also, there's no particular need for `nodeIntegration` in this window I don't think.

Reply:
👍 done!